Marlin/src/module/configuration_store.cpp

```diff
@@ -620,8 +620,6 @@ void MarlinSettings::postprocess() {
 
       #if ENABLED(Z_TRIPLE_ENDSTOPS)
         EEPROM_WRITE(endstops.z3_endstop_adj);   // 1 float
-      #else
-        EEPROM_WRITE(dummy);
       #endif
```

Comment:
We write placeholders for disabled features, and as long as there's parity between the read and the write, all should be well.